.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,17 @@
-## PLEASE READ
-
-Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).
-
-- [ ] Yes, I've run `yarn install`
-- [ ] No, does not apply to me
-
 ## Purpose of this pull request
 
-This pull request...
+This pull request... <!-- brief description here -->
 
-<!-- Enter the GitHub Issue number or the Jira project and number (e.g., SUMO-12345) -->
+<!-- Enter the GitHub Issue number or the Jira ticket number (e.g., SUMO-12345) -->
```

Comment:
Could you move this comment line about ticket numbers down to the section about tickets?